src/routes/NotFound/index.tsx

```diff
@@ -0,0 +1,12 @@
+import { Typography } from '@mui/material';
+import { Stack } from '@mui/system';
+
+const NotFound = (): JSX.Element => {
```

Comment:
This should have been created in the same commit that is used

Reply:
I wanted to separate the NotFound creating and the refactor of the routers. I should`ve done in another PR